e2e/__tests__/failureDetailsProperty.test.ts

```diff
@@ -0,0 +1,179 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as path from 'path';
+import {isJestCircusRun} from '@jest/test-utils';
+import runJest from '../runJest';
+
+const cleanAbsolutePaths = (stdout: string) =>
+  stdout.replace(
+    new RegExp(path.resolve(__dirname, '../..'), 'gu'),
+    '<rootDir>',
+  );
+
+test('that the failureDetails property is set', () => {
+  const {stdout} = runJest('failureDetails-property', ['tests.test.js']);
+
+  if (isJestCircusRun()) {
+    expect(cleanAbsolutePaths(stdout)).toMatchInlineSnapshot(`
+      "Custom reporter output:
+      [
+        [ { matcherResult: [Object] } ],
```

Comment:
why is this `[Object]` while jasmine actually prints the object?

Reply:
Because I didn't use `utils.inspect` (or `JSON.stringify`) 😄